.env.e2e.example

```
@@ -0,0 +1,5 @@
+NEXT_PUBLIC_API_BASE_URL=https://api.firstlegoleague.win
+E2E_API_BASE_URL=https://api.firstlegoleague.win
+E2E_ALLOW_REMOTE_WRITES=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The E2E_ALLOW_REMOTE_WRITES key should go before the E2E_API_BASE_URL key

(UnorderedKey)